### DISCUSSION TITLE
memory calculator - can I set the XMS value?

### DISCUSSION THREAD

**Discussion by heesuk-ahn** [paketo-buildpacks/feedback#31] [Q&A]
hi, all

I know that the memory calcurator calculates the current XMX value based on the container's resource request value.

In order to prevent GC from occurring frequently in the early stage of the JVM, 

I want to set XMX and XMS to the same value.

However, the `XMX` value is dynamically changed by the memory calcurator, so is there any function to set `XMS`?

**Comment by robdimsdale** [comment]
cc @paketo-buildpacks/java-buildpacks

**Reply by pivotal-david-osullivan** [reply]
There isn't a dynamic way to do this at the moment, it is currently marked as an enhancement. See here for the reasons and more info: https://github.com/paketo-buildpacks/libjvm/issues/99

As mentioned there, you can set the min value via `JAVA_TOOL_OPTIONS` manually once you determine the auto-calculated max. 

I hope this helps!
